hazelcast-wm/src/main/java/com/hazelcast/web/ClusteredSessionService.java

```diff
@@ -92,19 +89,21 @@
      * @param filterConfig   the filter config
      * @param properties     the properties
      * @param clusterMapName the cluster map name
-     * @param sessionTTL     the session tTL
      */
-    public ClusteredSessionService(FilterConfig filterConfig, Properties properties, String clusterMapName, String sessionTTL) {
+    public ClusteredSessionService(FilterConfig filterConfig, Properties properties, String clusterMapName) {
         this.filterConfig = filterConfig;
         this.properties = properties;
         this.clusterMapName = clusterMapName;
-        this.sessionTTL = sessionTTL;
+        init();
+    }
 
+    public void init() {
         try {
```

Comment:
It isn't part of this PR, but the code below is quite worrying because every exception thrown is swallowed:
* no stacktraces
* ClusteredSessionService can be left in an broken state 
* improper handling of InterruptedException?